CONTRIBUTING.md

```diff
@@ -1,49 +1,179 @@
+
+* `template=docs.md`
+* `template=other.md`
+
+### Requesting Reviews
+
+In order to accommodate the review process, the author of the PR must complete the author checklist
+(from the pull request template)
+to the best of their abilities before marking the PR as "Ready for Review". If you would like to
+receive early feedback on the PR, open the PR as a "Draft" and leave a comment in the PR indicating
+that you would like early feedback and tagging whoever you would like to receive feedback from.
+
+Codeowners are marked automatically as the reviewers.
+
+All PRs require at least two review approvals before they can be merged (one review might be acceptable in
+the case of minor changes to [docs](./.github/PULL_REQUEST_TEMPLATE/docs.md) or [other](./.github/PULL_REQUEST_TEMPLATE/other.md) changes that do not affect production code). Each PR template has a reviewers checklist that must be completed before the PR can be merged. Each reviewer is responsible
+for all checked items unless they have indicated otherwise by leaving their handle next to specific
+items. In addition, use the following review explanations:
+
+* `LGTM` without an explicit approval means that the changes look good, but you haven't thoroughly reviewed the reviewer checklist items.
+* `Approval` means that you have completed some or all of the reviewer checklist items. If you only reviewed selected items, you must add your handle next to the items that you have reviewed. In addition, follow these guidelines:
+    * You must also think through anything which ought to be included but is not
+    * You must think through whether any added code could be partially combined (DRYed) with existing code
+    * You must think through any potential security issues or incentive-compatibility flaws introduced by the changes
+    * Naming must be consistent with conventions and the rest of the codebase
+    * Code must live in a reasonable location, considering dependency structures (for example, not importing testing modules in production code, or including example code modules in production code).
+    * If you approve the PR, you are responsible for any issues mentioned here and any issues that should have been addressed after thoroughly reviewing the reviewer checklist items in the pull request template.
+* If you sat down with the PR submitter and did a pairing review, add this information in the `Approval` or your PR comments.
+* If you are only making "surface level" reviews, submit notes as a `comment` review.
+
+### Updating Documentation
+
+If you open a PR in ICS, it is mandatory to update the relevant documentation in `/docs`.
+
+## Dependencies
```

Comment:
To be update after https://github.com/cosmos/interchain-security/issues/801 is completed. See https://github.com/cosmos/cosmos-sdk/blob/main/CONTRIBUTING.md#dependencies for an example.

Reply:
cc @smarshall-spitzbart